templates/home.html

```diff
@@ -1,43 +1,137 @@
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Pastebin</title>
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/water.css@2/out/water.css"
-    />
-    <style>
-        body {
-            text-align: center;
-        }
-        
-        .paste {
-            display: inline-block;
-            margin: 0 auto;
-            text-align: center;
-            width: 50%;
-            max-width: 800px;
-        }
-
-        input {
-            margin-left: auto;
-            margin-right: auto;
-            text-align: center;
-        }
-    </style>
-  </head>
-  <body>
-    <h1>Pastebin</h1>
-    <nav>
-        <a href="/download">Download Files</a>
-    </nav>
-    <br>
-    <div>
-    %CONTENT%
-    </div>
-    <br>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Pastebin</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/water.css@2/out/water.css" />
+  <style>
+    * {
+      box-sizing: border-box;
+    }
+
+    body {
+      text-align: center;
+    }
+
+    .file {
+      position: relative;
+      display: inline-block;
+      margin: 0 auto;
+      border: 0px;
+      padding: 0px;
+      text-align: center;
+      width: 50%;
+      max-width: 800px;
+    }
+
+    #go {
+      float: left;
+      width: 10%;
+      height: 50px;
+      padding: 0px;
+      line-height: 50px;
+      text-align: center;
+    }
+
+    input {
+      margin-left: auto;
+      margin-right: auto;
+      text-align: center;
+    }
+
+    input[type="file"] {
+      width: 100%;
+      height: 50px;
+      padding: 0px;
+      content-visibility: hidden;
+    }
+
+    #label {
+      position: absolute;
+      top: 0;
+      left: 0;
+      border-radius: 5px;
+      width: 100%;
+      height: 50px;
+      line-height: 50px;
+      text-align: center;
+      user-select: none;
+    }
+
+    #label:hover {
+      border: 3px solid var(--focus);
+      line-height: 44px;
+    }
+
+    .paste {
+      width: 100%;
+      margin: auto;
+    }
+
+    #bin {
+      float: left;
+      margin-left: 25%;
+      margin-right: 10px;
+      text-align: center;
+      width: 40%;
+      height: 50px;
+      padding: 5px;
+      line-height: 40px;
+      max-width: 800px;
+      resize: none;
+      white-space: nowrap;
+      overflow-y: hidden;
+    }
+
+    #sep {
+      display: flex;
+      flex-direction: row;
+    }
+
+    #sep:before,
+    #sep:after {
+      content: "";
+      flex: 1 1;
+      border-bottom: 1px solid;
+      margin: auto;
+    }
+
+    #sep:before {
+      margin-right: 10px;
+      margin-left: 25%;
+    }
+
+    #sep:after {
+      margin-left: 10px;
+      margin-right: 25%;
+    }
```

Comment:
Move this css to a new file, and change the pastebin script to load the css with the html. This is too much for one file.

Reply:
Too many files = more load time